app/users/forms.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserChangeForm, UserCreationForm
+
+User = get_user_model()
+
+
+class CustomUserCreationForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ("email",)
+
+
+class CustomUserChangeForm(UserChangeForm):
+    class Meta:
+        model = User
+        fields = ("email",)
```

Comment:
Ideally, a user shouldn't be able to change their `email` as it serves as their unique username. 

For now, we can have password as a modifiable field. 
We can add other fields later, if required.